### PR TITLE
[C-1193] Fix oauth

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Ouch.

Simulator Screen Shot - iPhone 14 Pro - 2022-09-30 at 17 24 23

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Painfully. Also debugger mode does not work.

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by raymondjacobson** [comment]
Instagram continues to be a bit flaky, but this is a known issue already (unfortunately)

File: packages/mobile/src/screens/signon/ProfileAuto.tsx [hunk 1]
```diff
@@ -298,15 +301,18 @@ const ProfileAuto = ({ navigation, route }: ProfileAutoProps) => {
...
     if (twitterInfo) {
       dispatch(
         signOnActions.setTwitterProfile(
           twitterInfo.twitterId,
           twitterInfo.profile,
           twitterInfo.profile.profile_image_url_https
             ? {
-                uri: twitterInfo.profile.profile_image_url_https,
+                uri: twitterInfo.profile.profile_image_url_https.replace(
+                  /_(normal|bigger|mini)/g,
```
Comment (nicoback2) [inline packages/mobile/src/screens/signon/ProfileAuto.tsx:313]: OOC why is this needed?

**Review by nicoback2 — APPROVED** [review]
Awesome!!

File: packages/mobile/src/screens/signon/ProfileAuto.tsx (hunk 1 above)
Comment (raymondjacobson) [reply packages/mobile/src/screens/signon/ProfileAuto.tsx:314]: Added a comment. It's small by default and they don't expose the hd format one
